team-members/instructlab-admins.csv

```diff
@@ -25,3 +25,6 @@ pavolloffay,member
 tsisodia10,member
 AIMikav, member
 suppathak, member
+anyuzoey, member
+EoghanOConnor, member
+kevincogan, member
```

Comment:
It's the whitespace after `,` that was causing problems; it's considered to be part of the field. I'll push an update to this pr.